plugins/homepage/server/__init__.py

```diff
@@ -23,7 +23,12 @@
 from girder.utility import setting_utilities
 from girder.utility.model_importer import ModelImporter
 
-KEY = 'homepage.markdown'
+MARKDOWN = 'homepage.markdown'
+HEADER = 'homepage.header'
+SUBHEADER = 'homepage.subheader'
+WELCOME_TEXT = 'homepage.welcome_text'
```

Comment:
Should this be `welcome_template` ? Look like the final text is rendered differently based on the username, etc ...

Reply:
The text at the bottom (with bullet points) is independent from the "welcome text" (which appears directly above the bullet point text). Some documentation could probably help with explaining this.